package.json

```diff
@@ -55,10 +55,10 @@
     "clean": "rimraf dist node_modules coverage"
   },
   "dependencies": {
+    "cloudly-http": "0.0.50",
+    "cloudly-router": "0.0.24",
```

Comment:
I can't see why `cloudly-router` is a dependency. It is not used anywhere.
Remove it and make sure it works by testing the branch in a worker.